src/test/resources/application-stubbed.properties

```diff
@@ -0,0 +1,6 @@
+spring.kafka.bootstrap-servers = kafka-bootstrap:9092
+uk.gov.companieshouse.item-handler.error-consumer = false
+
+email.senderAddress = noreply@companieshouse.gov.uk
+email.producerTimeout = 30
+maximum.retries = 3
```

Comment:
Can we get away with using just this properties file?

Reply:
Unfortunately not